src/lang/german.txt

```diff
@@ -3,6 +3,8 @@
 ##isocode de_DE
 ##plural 0
 ##textdir ltr
+##numberformat 00.000.000.000.000.000.000
+##numberabbreviations 3=00.000.000.000.000.000{NBSP}t|6=00.000.000.000.000{NBSP}Mio|9=00.000.000.000{NBSP}Mrd|12=00.000.000{NBSP}Bio|15=00.000{NBSP}Qa|18=00{NBSP}Qi
```

Comment:
This is a weird mixture of default and adjusted values :)
Correct would be: Tsd., Mio., Mrd., Bio., Brd., Trill.

Reply:
The first four are coming from the current translation, the others are just the English ones. I'll add these to the PR.